src/sentry/auth/providers/oauth2.py

```diff
@@ -113,6 +113,9 @@ def dispatch(self, request: HttpRequest, helper) -> HttpResponse:
         if state != helper.fetch_state("state"):
             return helper.error(ERR_INVALID_STATE)
 
+        if code is None:
+            return helper.error("no code was provided")
```

Comment:
We should probably stick to the same format used in other errors (starting with a capital letter)
```suggestion
            return helper.error("No code was provided")
```